### PR TITLE
Use functions-framework-conformance action in conformance CI

### PR DESCRIPTION
**Pull request by hdp617**

### CODE REVIEW

File: .github/workflows/conformance.yml [hunk 1]
```diff
@@ -9,21 +9,44 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        runtime: ['nodejs10', 'nodejs12']
```
Comment (grant) [inline .github/workflows/conformance.yml:12]: Why not 14?

File: .github/workflows/conformance.yml [hunk 2]
```diff
@@ -9,21 +9,44 @@ jobs:
...
+    - name: Setup Go
+      uses: actions/setup-go@v2
       with:
-        go-version: '1.14'
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+        go-version: '1.13'
+    - name: Run HTTP conformance tests
+      uses: GoogleCloudPlatform/functions-framework-conformance/action@hdp617-patch-1
```
Comment (grant) [inline .github/workflows/conformance.yml:21]: Can we remove the patch?

**Review by grant — APPROVED** [review]

File: .github/workflows/conformance.yml (hunk 2 above)
Comment (hdp617) [reply .github/workflows/conformance.yml:21]: Yes, I'll remove it after I make another conformance release.

File: .github/workflows/conformance.yml (hunk 1 above)
Comment (hdp617) [reply .github/workflows/conformance.yml]: Done.

**Pull request merged**